resources/views/menus/browse.blade.php

```diff
@@ -107,7 +107,7 @@
         });
 
         $('td').on('click', '.delete', function (e) {
-            $('#delete_form')[0].action = '{{ route('voyager.'.$dataType->slug.'.destroy', ['menu' => '__menu']) }}'.replace('__menu', $(this).data('id'));
+            $('#delete_form')[0].action = '{{ route('voyager.'.$dataType->slug.'.destroy', ['id' => '__menu']) }}'.replace('__menu', $(this).data('id'));
```

Comment:
@MrCrayon, in response to your last comment, this is the line I was referring to.  You're using the `['x' => 'y']` syntax, which you agreed needed to be `widget` if the resource route was defined as `widgets`.  In this case, `id` shouldn't work, and instead should be the singular version of `$dataType->slug`, right? Or am I missing something?

Alternatively, we could remove the brackets altogether and just leave it as `route('....', '__menu')`, right?